### ISSUE TITLE
Farm Location Releases have placeholder data

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#548]
So I'm looking at LocationRelease info for The Farm (4188263703) and it appears that definitions have funky data that was perhaps left over from something else. Ie activityBubbleName often refers to bubbles in the Tower (or don't exist) like with the Clan Steward (842084602).

Wouldn't it make more sense to have a default bubble entry for this Destination and override any definitions tied to it to use that bubble?

**Comment by vthornheart-bng** [comment]
Yeah, in theory that would be nice - however, that's not how the data is set up.  This is unfortunately another case of remnant data being left behind.  If we had unlimited time and resources, I'd have a secondary cleanup process on our end that stripped that irrelevant data and put it into a form that's more friendly: however, I don't think we're going to get around to doing that until we have a feature of our own that uses this data.  Manual fixups on our side are inherently prone to breaking, and to doing so in ways that we might not notice right away: I try my best to avoid performing manual alteration of game data if we're not going to directly use it in our own features as a result.

Let's keep this on the future wishlist.  If we ever do end up using bubble data directly, we can loop back to this and decide if doing our own cleansing and restructuring of this data is worth the maintenance costs!
